src/modules/DressMe/DressMe.tsx

```diff
@@ -0,0 +1,81 @@
+import { Form, Formik } from "formik";
+import styled from "styled-components";
+import { Button, SectionContainer } from "../../components/General.styled";
+import PersonalDetails from "./PersonalDeatails/PersonalDeatails";
+import Sizes from "./Sizes";
+import TextArea from "./TextArea";
+import { validate } from "./validate";
+
+const DressMe = () => {
+  return (
+    <Container id="form-section">
+      <Title>Dress me!</Title>
+      <Formik
+        initialValues={{
+          email: "",
+          firstName: "",
+          lastName: "",
+          adress: "",
+          phone: "",
+          budget: "",
+          shirtSize: "",
+          pantsSize: "",
+          other: "",
+        }}
+        validate={validate}
+        onSubmit={(values) => {
+          console.log("onSubmit");
+          alert(JSON.stringify(values));
+        }}
+      >
+        {(formik) => {
```

Comment:
you're not using this formik argument, so you can remove it and put the StyledForm as a direct child of Formik, instead of using this render props structure

Reply:
I think I will need it to success page, I read about methods and props Formik has

Reply:
ok